cooperator/migrations/14.0.1.7.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    company = env["res.company"].browse(1)
+    if company:
+        company.cooperator_subscription_sequence_id = env.ref(
+            "cooperator.sequence_subscription"
+        )
+        company.cooperator_register_operation_sequence_id = env.ref(
+            "cooperator.sequence_register_operation"
+        )
```

Comment:
This will also set `company_id` on the sequence, right?

Reply:
Nope! Good spot.